lib/components/Navbar.js

```diff
@@ -46,52 +15,81 @@ class Navbar extends Component {
     store: PropTypes.object.isRequired
   };
 
-  render() {
-    // const styles = require('./Navbar.scss');
-    const {user, logout} = this.props;
+  componentDidMount() {
 
-    let iconMenuItems = [
-      {payload: '1', text: 'Profile'},
-      {payload: '2', text: 'Map'}
-    ];
+    const menu = document.getElementById('menu'),
+      WINDOW_CHANGE_EVENT = ('onorientationchange' in window) ? 'orientationchange' : 'resize';
 
-    const signInOutBtnStyles = {
-      backgroundColor: 'transparent',
-      signInOutBtnStyles
+    function toggleHorizontal() {
+      [].forEach.call(
+        document.getElementById('menu').querySelectorAll('.custom-can-transform'),
+        function (el) {
+          el.classList.toggle('pure-menu-horizontal');
```

Comment:
it has about the same support of flexbox and we are using that
http://caniuse.com/#search=flexbox
http://caniuse.com/#search=classList